test/exercism/track_stream_test.rb

```diff
@@ -139,6 +139,53 @@ def test_read_unread_status
     assert clock.unread?, "clock should be unread"
   end
 
+  def test_watermark_is_read_when_greater_than_a_view
```

Comment:
@kytrinyx decided to write this so I don't forget the details after sleeping. ;)